examples/workflow-standalone/app/diagram.html

```diff
@@ -1,38 +1,34 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="utf-8" />
-        <link rel="stylesheet" href="css/sprotty.css" />
-        <link rel="stylesheet" href="css/glsp-sprotty.css" />
-        <style>
-            html,
-            body {
-                width: 100%;
-                height: 100%;
-                overflow: hidden;
-            }
 
-            #sprotty {
-                position: absolute;
-                top: 0;
-                bottom: 0;
-                left: 0;
-                right: 0;
-            }
+<head>
+    <meta charset="utf-8" />
+    <style>
+        html,
+        body {
+            width: 100%;
+            height: 100%;
+            overflow: hidden;
+        }
 
-            svg {
-                width: 100%;
-                height: 100%;
-            }
-        </style>
-        <!-- support Microsoft browsers -->
```

Comment:
Was this removed on purpose or formatted away automatically? I wasn't able to test this on Ubuntu but I just wanted to make sure this is what we ant to do.

Reply:
Yes this was removed on purpose. It imported a non existing script ("lib/dom.js") which was needed previously to support Internet Explorer and earlier Versions of Edge but we removed the script a while back when IE reached EOL.